src/DynamoCore/Models/DynamoModel.cs

```diff
@@ -977,7 +977,16 @@ internal static void RaiseIExtensionStorageAccessWorkspaceOpened(WorkspaceModel
         {
             workspace.TryGetMatchingWorkspaceData(extension.UniqueId, out Dictionary<string, string> data);
             var extensionDataCopy = new Dictionary<string, string>(data);
-            extension.OnWorkspaceOpen(extensionDataCopy);
+
+            try
+            {
+                extension.OnWorkspaceOpen(extensionDataCopy);
+            }
+            catch (Exception ex)
+            {
+                Debug.WriteLine(ex.Message + " : " + ex.StackTrace);
```

Comment:
please use the logger and log these exceptions to the console - I think this will get stripped out in release builds.